languages/french.json

```diff
@@ -6,165 +6,93 @@
 		"File": "Fichier",
 		"OpenFolder": "Ouvrir un dossier",
 		"OpenFile": "Ouvrir un fichier",
-		"Save": "Enregistrer",
-		"SaveAs": "Enregistrer sous",
+		"Save": "Sauvegarder",

```

Comment:
"Enregistrer" and "Enregistrer sous" seem better than "Sauvegarder" and "Sauvegarder sous", to me, and it's the translation used in most software. (Lines 9 and 10)